### PR TITLE
Add The Isle egg (Evrima)

### PR DESCRIPTION
**Pull request by enozii**
### All Submissions:

* [ x] Have you followed the guidelines in our Contributing document?
* [ x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [ x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### New Server Submissions:

1. [x ] Does your submission pass tests (server is connectable)?
3. [x ] Have you added the server to the main README.md?
4. [x ] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

**Comment by Rage-GitHub** [comment]
Should probably update the egg name then to include evrima if it's not installing legacy, otherwise it can be misleading.

**Comment by enozii** [comment]
> Should probably update the egg name then to include evrima if it's not installing legacy, otherwise it can be misleading.

Fair enough, i might post the legacy egg at some point.

File: README.md [hunk 1]
```diff
@@ -237,6 +237,7 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [Subnautica: Nitrox Mod](game_eggs/steamcmd_servers/subnautica_nitrox_mod)
 * [Sven Co-op](game_eggs/steamcmd_servers/svencoop)
 * [The Forest](game_eggs/steamcmd_servers/the_forest)
+* [The Isle](game_eggs/steamcmd_servers/the_isle)
```
Comment (parkervcp) [inline README.md:240]: If there are 2 versions of the game make a sub-folder

so `game_eggs/steamcmd_servers/the_isle/evrima` and `game_eggs/steamcmd_servers/the_isle/legacy`

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (enozii) [reply README.md:240]: I've made the evrima sub folder, I'll make the legacy sub folder once I've double checked the egg.

Comment (parkervcp) [inline README.md:240]: This should probably have a subsection for `legacy` and `evrima`. Similar to the valheim `vanilla` and `plus mod` layout.

File: game_eggs/README.md [hunk 2]
```diff
@@ -154,6 +154,7 @@
 * [Sven Co-op](steamcmd_servers/svencoop)
 * [Team Fortress 2 Classic](steamcmd_servers/team_fortress_2_classic)
 * [The Forest](steamcmd_servers/the_forest)
+* [The Isle](steamcmd_servers/the_isle)
```
Comment (parkervcp) [inline game_eggs/README.md:157]: Same as my prior main readme comment. Showing evrima and legacy as separate sections under `The Isle`

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (enozii) [reply README.md:240]: Added subsection for Evrima

File: game_eggs/README.md (hunk 2 above)
Comment (enozii) [reply game_eggs/README.md:157]: Added subsection for Evrima

**Review by parkervcp — APPROVED** [review]

**Pull request merged**